public/content/lessons/2018/sphere-area/index.mdx

```diff
@@ -6,4 +6,421 @@ video: GNcFjFmqEc8
+
+> "The main idea will be to show a correspondence between these ring shadows [in the top hemisphere], and every 2nd ring on the sphere."
+
+So a reasonable correspondence might be to pair up each ring at $\theta$ with the ring at $2\theta$. That lets us cover alternating rings over the entire sphere, as hinted at above.
+
+So, what is the area of the ring at $2\theta$? Well, we can use our original ring area equation from question 1, but substitute $2\theta$ in place of $\theta$. That gives an area of $2\pi R^2 \sin(2\theta) \mathop{d\theta}$.
+
+We were hoping this would be double the area of the shadow of ring $\theta$, $4\pi R^2 \sin(\theta)\cos(\theta) \mathop{d\theta}$, but it doesn't really appear to be.
+
+This is where a trig identity can save the day:
+
+$$
+\sin(2\theta) = 2\sin(\theta)\cos(\theta)
+$$
+
+Using this substitution, we find that the ring at $2\theta$ has an area of $4\pi R^2 \sin(\theta)\cos(\theta) \mathop{d\theta}$, which is indeed double the area of the shadow of ring $\theta$.
+
+Therefore, the correspondence between ring $\theta$'s shadow and ring $2\theta$ has precisely the property we're looking for.
+
+</FreeResponse>
+
+### Question 4
+I said from the outset that there is a correspondence between all the shadows from the northern hemisphere, which make up a circle with radius $R$, and every other ring on the sphere. Use your answer to the last question to spell out exactly what that correspondence is.
+
+<Figure image="question-4.png" />
+
+<FreeResponse>
+
+Each shadow corresponds to one of the alternating rings of the sphere. The special property of this correspondence, spelled out in question 3, is that corresponding ring has an area double that of the shadow.
+
+This means that the alternating rings have a total area that is exactly double the total area of the shadows.
+
+</FreeResponse>
+
+### Question 5
+Bring it on home. Why does this imply that the area of the circle is exactly ¼ the surface area of the sphere, particularly as we consider thinner and thinner rings?
+
+<Figure image="question-5.png" />
+
+<FreeResponse>
+
+All together, the top-hemisphere ring shadows form a perfect circle of radius $R$ on the xy-plane. This means that the sum of all their areas is exactly $\pi R^2$.
+
+<Figure image="shadows-form-circle.png" video="shadows-form-circle.mp4" />
+
+And because of our special correspondence, we know that the total area of all the alternating rings on the sphere is exactly double the total shadow area. So adding up the areas of these rings of the sphere gives a surface area of $2\pi R^2$:
+
+<Figure
+  image="alternating-rings.png"
+  caption="The total surface area of these rings is $2\pi R^2."
+/>
+
+To get the full sphere surface area for *all* the rings (not just every-other ring), we double our result for an area of $4\pi R^2$.
```

Comment:
```suggestion
In the limit as these rings get ever thinner, the area of the alternating rings approaches half the total surface area. This is because each ring has approximately the same area as its immediate neighbors in that limiting case, so for example the sum of the areas of all the odd-numbered rings will equal the sum of the areas of the even-numbered rings. So to get the full surface area of the sphere, we double the total area of the alternating rings, which is $4\pi R^2$.
```